src/cdk/layout/breakpoints.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+// tslint:disable-next-line:variable-name
```

Comment:
Add explanation that we use PascalCase here because we want to treat this like an enum